client/src/styles/WorkflowField.scss

```diff
@@ -41,21 +41,14 @@
 
       .workflow-field-action-disabled {
         opacity: 0.35;
-        filter: Alpha(Opacity=35);
+        filter: alpha(Opacity=35);
         cursor: default;
 
         // These default styles are taken from .ss-ui-button (_forms.scss) so we don't end up with mousover/focus states on disabled buttons.
         // @todo These styles need to be dynamcally included from CMS, they are obviousely "hard-coded" for now.
         &.ui-state-hover,
         &.ui-state-focus {
           text-decoration: none;
-          background-color: #e6e6e6;
-          background: url("data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4gPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PGRlZnM+PGxpbmVhckdyYWRpZW50IGlkPSJncmFkIiBncmFkaWVudFVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgeDE9IjUwJSIgeTE9IjAlIiB4Mj0iNTAlIiB5Mj0iMTAwJSI+PHN0b3Agb2Zmc2V0PSIwJSIgc3RvcC1jb2xvcj0iI2ZmZmZmZiIvPjxzdG9wIG9mZnNldD0iMTAwJSIgc3RvcC1jb2xvcj0iI2Q5ZDlkOSIvPjwvbGluZWFyR3JhZGllbnQ+PC9kZWZzPjxyZWN0IHg9IjAiIHk9IjAiIHdpZHRoPSIxMDAlIiBoZWlnaHQ9IjEwMCUiIGZpbGw9InVybCgjZ3JhZCkiIC8+PC9zdmc+IA==");
-          background: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, $white), color-stop(100%, $grey-lightish));
-          background: -webkit-linear-gradient($white, $grey-lightish);
-          background: -moz-linear-gradient($white, $grey-lightish);
-          background: -o-linear-gradient($white, $grey-lightish);
-          background: -ms-linear-gradient($white, $grey-lightish);
           background: linear-gradient($white, $grey-lightish);
```

Comment:
linear-gradient is supported for all browsers we care about. Everything else here was old fallbacks